src/api/app/models/token/workflow.rb

```diff
@@ -12,21 +12,21 @@ def call(options)
     scm_extractor_payload = extractor.call
     yaml_file = Workflows::YAMLDownloader.new(scm_extractor_payload).call
     workflows = Workflows::YAMLToWorkflowsService.new(yaml_file: yaml_file, scm_extractor_payload: scm_extractor_payload, token: self).call
+    raise Token::Errors::InvalidWorkflowStepDefinition, 'Invalid workflow step definition' if
```

Comment:
This error should be raised in `Workflow`. I would raise this error when workflows are initialized, so in the `initialize` method of the `Workflow` class. In the error message, we should also use the `errors` method from `Workflow` to help users identify what needs to be done. `Invalid workflow step definition` doesn't help much.